bin/development-server.js

```diff
@@ -47,12 +43,14 @@ app.use(webpackDevMiddleware(compiler, {
   stats: { colors: true }
 }));
 
-if(feature.isEnabled("hotReloading")) {
+if (feature.isEnabled("hotReloading")) {
   app.use(webpackHotMiddleware(compiler));
+} else {
+  console.log("Hot Reloading can be enabled by adding " +
+  "\"hotReloading\": true to your local.json config");
```

Comment:
I don't think we want to message for every feature, but hot reloading is an important one so I wanted to include this.  At first I went with trying to link to the docs but it is simpler to just offer the solution.